src/Blocks/components/file/components/file-options.js

```diff
@@ -30,146 +30,144 @@ export const FileOptions = (attributes) => {
 	const fileIsDisabled = checkAttr('fileIsDisabled', attributes, manifest);
 
 	return (
-		<>
-			<PanelBody title={__('File', 'eightshift-forms')}>
-				<Section icon={icons.options} label={__('General', 'eightshift-forms')}>
-					<NameField
-						value={fileName}
-						attribute={getAttrKey('fileName', attributes, manifest)}
-						disabledOptions={fileDisabledOptions}
-						setAttributes={setAttributes}
-						type={'file'}
-						isChanged={isNameChanged}
-						setIsChanged={setIsNameChanged}
-					/>
-				</Section>
-
-				<FieldOptions
-					{...props('field', attributes, {
-						fieldDisabledOptions: fileDisabledOptions,
-					})}
+		<PanelBody title={__('File', 'eightshift-forms')}>
+			<Section icon={icons.options} label={__('General', 'eightshift-forms')}>
+				<NameField
+					value={fileName}
+					attribute={getAttrKey('fileName', attributes, manifest)}
+					disabledOptions={fileDisabledOptions}
+					setAttributes={setAttributes}
+					type={'file'}
+					isChanged={isNameChanged}
+					setIsChanged={setIsNameChanged}
+				/>
+			</Section>
+
+			<FieldOptions
+				{...props('field', attributes, {
+					fieldDisabledOptions: fileDisabledOptions,
+				})}
+			/>
+
+			<FieldOptionsLayout
+				{...props('field', attributes, {
+					fieldDisabledOptions: fileDisabledOptions,
+				})}
+			/>
+
+			<Section icon={icons.checks} label={__('Validation', 'eightshift-forms')}>
+				<IconToggle
+					icon={icons.fieldRequired}
+					label={__('Required', 'eightshift-forms')}
+					checked={fileIsRequired}
+					onChange={(value) => setAttributes({ [getAttrKey('fileIsRequired', attributes, manifest)]: value })}
+					disabled={isOptionDisabled(getAttrKey('fileIsRequired', attributes, manifest), fileDisabledOptions)}
+				/>
+
+				<TextControl
+					label={<IconLabel icon={icons.fileType} label={__('Accepted file types', 'eightshift-forms')} />}
+					value={fileAccept}
+					help={__('Separate items with a comma.', 'eightshift-forms')}
+					placeholder='e.g. .jpg,.png,.pdf'
```

Comment:
This should also get a localized label because of the _e.g._

Reply:
done